README.md

```diff
@@ -50,6 +50,24 @@ npm install
 npm run dev
 ```
 
+### To use Auth0
+
+- Use  `wallora.test@gmail.com` to login to auth0 [website](https://auth0.auth0.com/u/login/) using google. 
+- Copy `.env.local.sample` to `.env.local` and populate the fields from the app you see in auth0 post login.
+- After this when you run the dev server and try to login you should be able to see the auth0 login flow.
+
+Reference: https://auth0.com/docs/quickstart/webapp/nextjs
```

Comment:
this should have ideally been added as part of Auth0 integration. Will add it in this PR

Reply:
as per feedback was asked to add it. Since this isn't any feature, bug etc. Adding it with this PR itself.